datafusion/expr/src/lib.rs

```diff
@@ -16,6 +16,8 @@
 // under the License.
 // Make cheap clones clear: https://github.com/apache/datafusion/issues/11143
 #![deny(clippy::clone_on_ref_ptr)]
+// TODO When the deprecated trait AggregateExt is removed, remove this unstable feature.
```

Comment:
It seems like this feature requires nightly rust, which isn't possible for DataFusion I don't think. I'll push a commit to this PR that removes the attempt at backwards compatibility and just updated the description